engine/reference/commandline/build.md

```diff
@@ -94,11 +96,9 @@ Build Syntax Suffix             | Commit Used           | Build Context Used
 
 If you pass an URL to a remote tarball, the URL itself is sent to the daemon:
 
-Instead of specifying a context, you can pass a single Dockerfile in the `URL`
-or pipe the file in via `STDIN`. To pipe a Dockerfile from `STDIN`:
-
```

Comment:
You are probably right, I should do a follow-on commit that re-replaces the reference docs with what is currently in engine.